sgp30/sgp30.c

```diff
@@ -457,21 +327,16 @@ int16_t sgp30_measure_raw() {
  * Return:      STATUS_OK on success, an error code otherwise
  */
 int16_t sgp30_read_raw(uint16_t *ethanol_raw_signal, uint16_t *h2_raw_signal) {
-    const struct sgp_profile *profile;
     int16_t ret;
+    uint16_t words[SGP30_CMD_RAW_MEASURE_WORDS];
 
-    profile = sgp30_get_profile_by_number(PROFILE_NUMBER_MEASURE_RAW_SIGNALS);
-    if (profile == NULL)
-        return STATUS_FAIL;
-
-    ret = read_measurement(profile);
-    if (ret != STATUS_OK)
-        return ret;
+    ret = sensirion_i2c_read_words(SGP30_I2C_ADDRESS, words,
+                             SGP30_CMD_RAW_MEASURE_WORDS);
```

Comment:
Did this pass the format check? It looks odd.

Reply:
No it didn't. Fixed now